R/fiach.R

```diff
@@ -263,7 +279,7 @@ fiach <-function(input,t,tr,rp=NULL,maxgap=1,freq=128,nMads = 1.96,defaultMask=T
   scrubbed.brain<-filt.brain
   scrubbed.brain[scrub.inds]<-brain.meds[scrub.inds[,2]]
   output.brain.mat<-hp.mat                                                                    
-  #output.brain.mat[,which(mask.mat==FALSE)]<-0                                           
+  output.brain.mat[,which(mask.mat==FALSE)]<-0                                           
```

Comment:
I think There should probably be a check here to see if the user has used an external mask before zeroing out data.